config/config_helpers_test.go

```diff
@@ -560,120 +313,57 @@ func TestResolveEnvInterpolationConfigString(t *testing.T) {
 			nil,
 			terragruntOptionsForTestWithEnv(t, "/root/child/"+DefaultTerragruntConfigPath, map[string]string{"TEST_ENV_TERRAGRUNT_OTHER": "SOMETHING"}),
 			"foo//bar",
-			nil,
-		},
-		{
-			`foo/${get_env(    "TEST_ENV_TERRAGRUNT_HIT"   ,   ""   )}/bar`,
-			nil,
-			terragruntOptionsForTestWithEnv(t, "/root/child/"+DefaultTerragruntConfigPath, map[string]string{"TEST_ENV_TERRAGRUNT_OTHER": "SOMETHING"}),
-			"foo//bar",
-			nil,
-		},
-		{
-			`foo/${get_env("TEST_ENV_
-TERRAGRUNT_HIT","")}/bar`,
-			nil,
-			terragruntOptionsForTestWithEnv(t, "/root/child/"+DefaultTerragruntConfigPath, map[string]string{"TEST_ENV_TERRAGRUNT_OTHER": "SOMETHING"}),
 			"",
-			InvalidInterpolationSyntax(`${get_env("TEST_ENV_
-TERRAGRUNT_HIT","")}`),
 		},
 		{
-			`foo/${get_env("TEST_ENV_TERRAGRUNT_HIT","DEFAULT")}/bar`,
+			`iam_role = "foo/${get_env("TEST_ENV_TERRAGRUNT_HIT","DEFAULT")}/bar"`,
 			nil,
 			terragruntOptionsForTestWithEnv(t, "/root/child/"+DefaultTerragruntConfigPath, map[string]string{"TEST_ENV_TERRAGRUNT_OTHER": "SOMETHING"}),
 			"foo/DEFAULT/bar",
-			nil,
-		},
-		{
-			`foo/${get_env(    "TEST_ENV_TERRAGRUNT_HIT      "   ,   "default"   )}/bar`,
-			nil,
-			terragruntOptionsForTestWithEnv(t, "/root/child/"+DefaultTerragruntConfigPath, map[string]string{"TEST_ENV_TERRAGRUNT_HIT": "environment hit  "}),
-			"foo/environment hit  /bar",
-			nil,
-		},
-		{
-			`foo/${get_env("TEST_ENV_TERRAGRUNT_HIT","default")}/bar`,
-			nil,
-			terragruntOptionsForTestWithEnv(t, "/root/child/"+DefaultTerragruntConfigPath, map[string]string{"TEST_ENV_TERRAGRUNT_HIT": "HIT"}),
-			"foo/HIT/bar",
-			nil,
-		},
-		{
-			// Unclosed quote
-			`foo/${get_env("TEST_ENV_TERRAGRUNT_HIT}/bar`,
-			nil,
-			terragruntOptionsForTest(t, "/root/child/"+DefaultTerragruntConfigPath),
 			"",
-			InvalidInterpolationSyntax(`${get_env("TEST_ENV_TERRAGRUNT_HIT}`),
-		},
-		{
-			// Unclosed quote and interpolation pattern
-			`foo/${get_env("TEST_ENV_TERRAGRUNT_HIT/bar`,
-			nil,
-			terragruntOptionsForTest(t, "/root/child/"+DefaultTerragruntConfigPath),
-			`foo/${get_env("TEST_ENV_TERRAGRUNT_HIT/bar`,
-			nil,
 		},
 	}
 
 	for _, testCase := range testCases {
-		actualOut, actualErr := ResolveTerragruntConfigString(testCase.str, testCase.include, testCase.terragruntOptions)
-		if testCase.expectedErr != nil {
-			assert.True(t, errors.IsError(actualErr, testCase.expectedErr), "For string '%s' include %v and options %v, expected error %v but got error %v and output %v", testCase.str, testCase.include, testCase.terragruntOptions, testCase.expectedErr, actualErr, actualOut)
-		} else {
-			assert.Nil(t, actualErr, "For string '%s' include %v and options %v, unexpected error: %v", testCase.str, testCase.include, testCase.terragruntOptions, actualErr)
-			assert.Equal(t, testCase.expectedOut, actualOut, "For string '%s' include %v and options %v", testCase.str, testCase.include, testCase.terragruntOptions)
-		}
+		testCase := testCase
```

Comment:
NIT: We usually add the boilerplate comment about this being done to bring the var into the scope of the forloop.

Reply:
Fixed: https://github.com/gruntwork-io/terragrunt/pull/731/commits/9f19fd96d8c8301fb6078b1d86aac115a6bea00b